community-operators/jaeger/jaeger.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: jaeger
+channels:
+- name: alpha
+  currentCSV: jaeger-operator.v1.8.2
```

Comment:
If this is the convention we are following for naming csv, can we please add it to the documentation if we haven't done so already. Naming convention for package, csv, channel etc. and then getting them verified by OLM team.